Team Score Changes: Functionality to edit and see team scores on teams and layout page.

ria-dharmani opened the pull request:
* Created a new table, `StaffTeamFeedback`
* A new component, teamFeedback.tsx: To render the table in admin and nonadmin views
* Now the table can be seen above the peer feedback tables.

ria-dharmani committed:
Team Score Changes

ria-dharmani committed:
Works

ria-dharmani committed:
working updates

theantariksha commented:
Just tested it, looks like the score update works now in both tables!
On a side note, did you intend to merge the .vscode debug configuration file?

theantariksha reviewed (commented):
Quick testing shows yesterday's issue is resolved, thanks!

ria-dharmani committed:
Delete .vscode/launch.json

ria-dharmani commented:
> Just tested it, looks like the score update works now in both tables! On a side note, did you intend to merge the .vscode debug configuration file?
My bad, removed it now

theantariksha commented:
> > Just tested it, looks like the score update works now in both tables! On a side note, did you intend to merge the .vscode debug configuration file?
> > My bad, removed it now

No problem, just to let you know there is one more launch.json and the .DS_Store file, you probably want to remove these as well.

ria-dharmani committed:
.gitignore updates

ria-dharmani committed:
Delete launch.json

ria-dharmani commented:
> > > Just tested it, looks like the score update works now in both tables! On a side note, did you intend to merge the .vscode debug configuration file?
> > > My bad, removed it now
> 
> No problem, just to let you know there is one more launch.json and the .DS_Store file, you probably want to remove these as well.

Oh yes, I missed this before, I deleted them now

theantariksha reviewed (approved):
Thank you for the changes!
